packager.sh

```diff
@@ -548,13 +509,6 @@ _pkgtest_innervm_run () {
     # XXX: should the --upgrade-db command go in the postint script?
     ssh $lxc_ip "set -e; oq-engine --upgrade-db --yes"
 
-    # run celeryd daemon
-    ssh $lxc_ip "cd /usr/share/openquake/engine ; celeryd >/tmp/celeryd.log 2>&1 3>&1 &"
```

Comment:
I would keep this part as option, controlled by an ENV var so celery can be used, on demand, to run the demos.

A sed to change the openquake.cfg must be add to. I will do.